pkg/kubelet/certificate/certificate_manager.go

```diff
@@ -160,9 +148,16 @@ func (m *manager) shouldRotate() bool {
 	m.certAccessLock.RLock()
 	defer m.certAccessLock.RUnlock()
 	notAfter := m.cert.Leaf.NotAfter
-	total := notAfter.Sub(m.cert.Leaf.NotBefore)
-	remaining := notAfter.Sub(time.Now())
-	return remaining < 0 || uint(remaining*100/total) < m.shouldRotatePercent
+	totalDuration := float64(notAfter.Sub(m.cert.Leaf.NotBefore))
+
+	// Use some jitter to set the rotation threshold so each node will rotate
+	// at approximately 70-90% of the total lifetime of the certificate.  With
+	// jitter, if a number of nodes are added to a cluster at approximately the
+	// same time (such as cluster creation time), they won't all try to rotate
+	// certificates at the same time for the rest of the life of the cluster.
+	jitteryDuration := wait.Jitter(time.Duration(totalDuration*0.2), 1.0) + time.Duration(totalDuration*0.5)
```

Comment:
Shoudn't it be: s/0.5/0.7/ ?
I think that now it is producing together a number in 0.5-0.7 (random from [0..0.2] + 0.5).